### DISCUSSION TITLE
Public Sans loads Public Sans Thin

### DISCUSSION THREAD

**Discussion by koji** [fontsource/fontsource#661] [Q&A] [answered]
Hi there,

I'm trying to use Public Sans but the font loaded is `Public Sans Thin`.
Does anyone know a way to load `Public Sans`?

Any suggestions are greatly appreciated 🙏 

https://codesandbox.io/p/sandbox/serene-bird-2k1x7n?file=README.md
![Screenshot 2023-04-06 at 7 12 59 PM](https://user-images.githubusercontent.com/474225/230510219-bfa8bbde-2dc2-4de9-ae91-959ef4696707.png)

**Comment by ayuhito** [comment, marked as answer]
You have only imported `font-weight: 400` of Public Sans in your `main.tsx`. Adding additional imports such as `import "@fontsource/open-sans/700.css"` would fix the text as the default weight of the `h2` element is 700.

Reference: https://fontsource.org/docs/getting-started (I agree the documentation doesn't make that obvious. I do have a rewrite in progress though!)

**Reply by koji** [reply]
@ayuhito Thank you for your response.
I could import fonts properly.
